test_browser/puppeteer/puppeteer_deno.ts

```diff
@@ -0,0 +1,18 @@
+// BUG:
+// PUPPETEER_PRODUCT=chrome deno run -A --unstable puppeteer_deno.ts
+// Uncaught PermissionDenied: Permission denied (os error 13)
+// https://github.com/lucacasonato/deno-puppeteer/issues/72
+
+import puppeteer from "https://deno.land/x/puppeteer@16.2.0/mod.ts"
+// import puppeteer from "npm:puppeteer"
```

Comment:
I see there's a bug with the deno.land version of puppeteer; does the `npm:` specifier work?

It's definitely a good first step to run it at all, but it would be ideal if it could work without needing node.

Reply:
Can you please share the full error?

Reply:
fixed -> https://github.com/lucacasonato/deno-puppeteer/issues/51